### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jofftiquez**

### CODE REVIEW

**Comment by prmsrswt** [comment]
Add a section for setting up Development environment. This will have using `yarn` to install and add dependencies.

**Comment by jofftiquez** [comment]
Do you have a more accurate details rather than "etc"? Thanks.

**Comment by prmsrswt** [comment]
Something like this should work

```
## Development Workflow

- Run `yarn` to install dependencies.
- You will need a running MongoDB server.
- Create a file called `.env` using `.env.sample` as a template.
- Start the server using `yarn start`.

### Some considerations
- We do not use npm. Instead this project uses yarn.
- Try to follow the coding style of the project. Note that in future we will use `eslint` and `prettier`.
```

**Comment by jofftiquez** [comment]
Updated. Thanks.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,3 +7,25 @@ All sorts of contributions are welcome.
...
+
+**Dependencies**
+
+We strictly use **Yarn**. Please do not use **NPM**.
+
+**Database**
+
+You will need a running **MongoDB** server.
+
+**Environment Variavles**
```
Comment (prmsrswt) [inline CONTRIBUTING.md]: Minor typo here.

**Comment by prmsrswt** [comment]
There was a minor typo so I took the liberty and fixed it.

Thanks for the contribution. :tada:

**Pull request merged**